.github/workflows/ci-artifacts.yml

```diff
@@ -43,6 +43,46 @@ jobs:
         with:
           name: minimal-sdk
           path: artifacts/git-sdk-aarch64-minimal.tar.gz
+      - name: debug using tmate (detached)
```

Comment:
There is currently no Msys2 https://github.com/actions/partner-runner-images/blob/main/images%2Farm-windows-11-image.md

Reply:
Right, I found that out by looking around via `tmate` ;-)

Reply:
I meant to confirm your suspicions from the commit message, not to object to the patch.

Reply:
@rimrul I guess you're right, the hosted runners are building off of the partner images. I thought that they might be different (the _are_ different for x86_64 runners, after all) and was looking at the link in the job logs that is _supposed_ to point to the list of included software [but that link is broken](https://github.com/orgs/community/discussions/155713#discussioncomment-12840603).

Reply:
Yes, the inventory links seem like a bug (https://github.com/actions/partner-runner-images/issues/76)